src/utils/context.ts

```diff
@@ -113,3 +113,11 @@ export function sendSlack(message: string): boolean {
   });
   return true;
 }
+
+export function isRunningInKubernetesPod(): boolean {
+  // Check if the standard Kubernetes environment variables are set
+  return (
+    process.env.KUBERNETES_SERVICE_HOST !== undefined &&
+    process.env.KUBERNETES_SERVICE_PORT !== undefined
```

Comment:
where are these in the docs/readme?

Reply:
These are [automatically passed in by Kubernetes](https://kubernetes.io/docs/concepts/containers/container-environment/) when it starts the container (no configuration required on behalf of the watch-tower).